neon_tts_plugin_coqui_ai/__init__.py

```diff
@@ -37,14 +37,31 @@
     from ovos_plugin_manager.templates.tts import TTS, TTSValidator
 from neon_utils.metrics_utils import Stopwatch
 
-
-class TemplateTTS(TTS):  # TODO: Replace 'Template' with TTS name
-    def __init__(self, lang="en-us", config=None):
-        config = config or get_neon_tts_config().get("tts_module_name", {})  # TODO: Update name
-        super(TemplateTTS, self).__init__(lang, config, TemplateTTSValidator(self),
-                                          audio_ext="mp3",  # TODO: Specify output audio format
-                                          ssml_tags=["speak"])  # TODO: Specify valid SSML tags
-        # TODO: Optionally define any class parameters
+from huggingface_hub import snapshot_download
+
+from TTS.utils.manage import ModelManager
+from TTS.utils.synthesizer import Synthesizer
+
+
+class CoquiTTS(TTS):
+    langs = {
+        "en": {
```

Comment:
Are there other language models out there we can use (or even better a maintained list we can reference)

Reply:
We can add all models that Coqui has already, `tts --list_models`
But not all of them has the best quality 
For now I have added best English model and Polish model for the purposes of the demo

Reply:
Created an issue to address this later (https://github.com/NeonGeckoCom/neon-tts-plugin-coqui-ai/issues/2)